src/app/booking/Intro.js

```diff
@@ -0,0 +1,95 @@
+"use client";
+
+import { Box, Container } from "@mui/material";
+import Typography from "@mui/material/Typography";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { useTheme } from "@mui/material/styles";
+import Grid from "@mui/material/Grid";
+import DemoStepper from "@/app/(homepage)/DemoStepper";
+
+function IntroText(props) {
+
+  const STEPS = [
+    "Select activities",
+    "Activities are added to your child’s calendar",
+    "Pay for all bookings in one go",
+  ];
+
+  return (
+    <Box textAlign="center" {...props}>
+      <Typography my={3} variant="h1">
+        Discover, book and track activities without clicking,
```

Comment:
Orange text above should be here.

Reply:
Ok, I'll add it. This inscription was added after I created the Intro (screenshot from slack from figma)

![image](https://github.com/digital-brother/ooscca-fe/assets/49399157/dad7ed28-e73c-4e54-a9fa-ad44a7c92e0f)